3.5/indexing-ttl.md

```diff
@@ -86,6 +87,29 @@ It should be intuitive to see that the `expireDate` can be differently per docum
 This allows mixing of documents with different expiration periods by calculating their
 expiration dates differently in the client application.
 
+### Format of date/time values
+
+The expiration date time values can be specified either as a numeric timestamp, containing
+the number of milliseconds since January 1st 1970 (commonly referred to as Unix timestamp), 
+or as a date/time string in base format `YYYY-MM-DDTHH:MM:SS`, with optional millisecond 
+precision and an optional timezone offset. The timezone offset can be specified as either 
+`Z` (Zulu/UTC time) or as a deviation from UTC time in hours and minutes (i.e. `+HH:MM` or `-HH:MM`).
+
+Valid example date string values are:
+
+* `2019-05-27`: May 27th 2019, 00:00:00 UTC time
+* `2019-05-27T21:20:00`: May 27th 2019, 21:20:00 UTC time
+* `2019-05-27T21:20:00Z`: May 27th 2019, 21:20:00 UTC time
+* `2019-05-27T21:20:00.123Z`: May 27th 2019, 21:20:00.123 UTC time
+* `2019-05-27T21:20:00.123+01:30`: May 27th 2019, 21:20:00.123, +01:30 offset from UTC time
+* `2019-05-27T21:20:00.123-02:00`: May 27th 2019, 21:20:00.123, -02:00 offset from UTC time
+
+Using an invalid date string value in a document's TTL index attribute will prevent the document
+from being inserted into the TTL index, so it will not be expired and removed automatically.
```

Comment:
neither be expired nor be removed automatically

Reply:
``Using an invalid date string value in a document's TTL index attribute will prevent the document
from being inserted into the TTL index``

Will this raise an error or it will be accepted as a successful operation?

Also, what if user tries to insert a document that have an already expired valid datetime string?
a) ArangoDB doesn't even insert it and returns no error.
b) ArangoDB doesn't insert it returns and error.
c) ArangoDB inserts the document and lets background job remove it on next iteration?

Reply:
No error will be raised if a document is inserted with an invalid date string or with a date string value in the past.
Documents with invalid date string values will not be indexed, so they will not expire and not be removed automatically.
Documents with valid date string values that are in the past should be removed on the next run of the TTL background thread.